specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-03-30-preview/automanage.json

```diff
@@ -0,0 +1,421 @@
+            "description": "Parameters supplied to the create or update configuration profile assignment intent."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/VmNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created. The Automanage configuration profile assignment intent for the VM was created successfully.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationProfileAssignmentIntent"
+            }
+          },
+          "200": {
+            "description": "OK. The configuration profile assignment intent for the VM was updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationProfileAssignmentIntent"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ]
+      },
+      "get": {
+        "tags": [
+          "ConfigurationProfileAssignmentIntents"
+        ],
+        "operationId": "ConfigurationProfileAssignmentIntents_Get",
+        "description": "Get information about a configuration profile assignment intent",
+        "x-ms-examples": {
+          "Get a configuration profile assignment intent": {
+            "$ref": "./examples/getConfigurationProfileAssignmentIntent.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "configurationProfileAssignmentIntentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The configuration profile assignment intent name."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/VmNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Missing 404

Reply:
all errors should fall through to "default"

Reply:
Thank you both for reviewing this PR!

@Bpoe, Based on @pilor comment, I assume the current default is good enough. I will not change.